### ISSUE TITLE
deliver download_metadata always prompts to overwrite the folder

### ISSUE THREAD

**Issue by barnaclebarnes** [fastlane/fastlane#6875]
### New Issue Checklist

- [* ] Updated fastlane to the latest version
- [* ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

running `deliver download_metadata` always prompts to overwrite the folder even when it doesn't exist. This means that you cannot run the command unattended. It will always ask to overwrite the folder.

##### Complete output when running fastlane, including the stack trace and command used

```
➜  mytours-ios git:(experiment/fastlane) ✗ deliver download_metadata
[17:33:31]: Successfully loaded '/Users/barnacle/Documents/mytours/mytours-ios/Deliverfile' 📄

+------------------+-----------------------------+
|      Detected Values from './Deliverfile'      |
+------------------+-----------------------------+
| app_identifier   | com.example.app1036      |
| username         | xxx@example.com |
| metadata_path    | downloads/metadata/1036     |
| screenshots_path | downloads/screenshots/1036  |
+------------------+-----------------------------+

[17:33:31]: Login to iTunes Connect (xxx@example.com)
[17:33:33]: Login successful

+-----------------------+-----------------------------+
|               deliver 1.15.0 Summary                |
+-----------------------+-----------------------------+
| app_identifier        | com.mytoursapp.app1036      |
| username              | xxx@example.com |
| metadata_path         | downloads/metadata/1036     |
| screenshots_path      | downloads/screenshots/1036  |
| skip_binary_upload    | false                       |
| skip_screenshots      | false                       |
| skip_metadata         | false                       |
| force                 | false                       |
| submit_for_review     | false                       |
| automatic_release     | false                       |
| overwrite_screenshots | false                       |
+-----------------------+-----------------------------+

Do you want to overwrite existing metadata on path '/Users/barnacle/Documents/mytours/mytours-ios/downloads/metadata/1036'? (y/n)
```
### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                              |
| ------------------- | -------------------------------------------------- |
| OS                  | 10.12.1                                            |
| Ruby                | 2.3.1                                              |
| Bundler?            | true                                               |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/        |
| Xcode Version       | 8.1                                                |
| Git                 | git version 2.10.1                                 |
| Installation Source | /Users/barnacle/.rbenv/versions/2.3.1/bin/fastlane |
| Host                | Mac OS X 10.12.1 (16B2555)                         |
| Ruby Lib Dir        | /Users/barnacle/.rbenv/versions/2.3.1/lib          |
| OpenSSL Version     | OpenSSL 1.0.2j  26 Sep 2016                        |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem                 | Version | Update-Status |
| ------------------- | ------- | ------------- |
| credentials_manager | 0.16.2  | ✅ Up-To-Date  |
| fastlane_core       | 0.53.0  | ✅ Up-To-Date  |
| spaceship           | 0.37.0  | ✅ Up-To-Date  |
| cert                | 1.4.3   | ✅ Up-To-Date  |
| deliver             | 1.15.0  | ✅ Up-To-Date  |
| frameit             | 3.0.0   | ✅ Up-To-Date  |
| gym                 | 1.12.0  | ✅ Up-To-Date  |
| sigh                | 1.11.2  | ✅ Up-To-Date  |
| match               | 0.10.0  | ✅ Up-To-Date  |
| pem                 | 1.4.0   | ✅ Up-To-Date  |
| pilot               | 1.12.0  | ✅ Up-To-Date  |
| produce             | 1.3.0   | ✅ Up-To-Date  |
| scan                | 0.14.1  | ✅ Up-To-Date  |
| screengrab          | 0.5.5   | ✅ Up-To-Date  |
| snapshot            | 1.16.3  | ✅ Up-To-Date  |
| supply              | 0.7.1   | ✅ Up-To-Date  |
| fastlane            | 1.107.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.2        |
| bundler                   | 1.13.6       |
| io-console                | 0.4.6        |
| rake                      | 11.3.0       |
| CFPropertyList            | 2.3.3        |
| i18n                      | 0.7.0        |
| json                      | 1.8.3        |
| minitest                  | 5.9.1        |
| thread_safe               | 0.3.5        |
| tzinfo                    | 1.2.2        |
| activesupport             | 4.2.7.1      |
| public_suffix             | 2.0.3        |
| addressable               | 2.5.0        |
| babosa                    | 1.0.2        |
| byebug                    | 9.0.6        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander                 | 4.4.0        |
| security                  | 0.1.3        |
| credentials_manager       | 0.16.2       |
| excon                     | 0.54.0       |
| gh_inspector              | 1.0.2        |
| multi_json                | 1.12.1       |
| plist                     | 3.1.0        |
| rubyzip                   | 1.1.7        |
| terminal-table            | 1.4.5        |
| fastlane_core             | 0.53.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.9.2        |
| unf_ext                   | 0.0.7.2      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20161021 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.10.0       |
| fastimage                 | 1.6.8        |
| multi_xml                 | 0.5.5        |
| claide                    | 1.0.1        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| cocoapods-core            | 1.0.0        |
| cocoapods-deintegrate     | 1.0.1        |
| cocoapods-downloader      | 1.1.2        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.0.0        |
| netrc                     | 0.7.8        |
| cocoapods-trunk           | 1.1.1        |
| cocoapods-try             | 1.1.0        |
| escape                    | 0.0.4        |
| fourflusher               | 0.3.2        |
| molinillo                 | 0.4.5        |
| nanaimo                   | 0.2.2        |
| xcodeproj                 | 1.4.1        |
| cocoapods                 | 1.0.0        |
| coderay                   | 1.1.1        |
| dotenv                    | 2.1.1        |
| mini_magick               | 4.5.1        |
| rouge                     | 1.11.1       |
| xcpretty                  | 0.2.4        |
| net-ssh                   | 3.2.0        |
| net-sftp                  | 2.1.2        |
| krausefx-shenzhen         | 0.14.10      |
| slack-notifier            | 1.5.1        |
| xcpretty-travis-formatter | 0.0.4        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| logging                   | 2.1.0        |
| memoist                   | 0.15.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.1        |
| httpclient                | 2.8.2.4      |
| hurley                    | 0.2          |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.0.15       |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| google-api-client         | 0.9.19       |
| terminal-notifier         | 1.7.1        |
| word_wrap                 | 1.0.0        |
| xcode-install             | 2.0.8        |
| httparty                  | 0.14.0       |
| method_source             | 0.8.2        |
| mini_portile2             | 2.1.0        |
| nokogiri                  | 1.6.8.1      |
| slop                      | 3.6.0        |
| pry                       | 0.10.4       |
| pry-byebug                | 3.4.0        |
| zip                       | 2.0.2        |
</details>

*generated on:* **2016-11-06**
</details>

**Comment by TKBurner** [comment]
@barnaclebarnes What happens if you set `force: true`? That should prevent this.

**Comment by barnaclebarnes** [comment]
@TKBurner I tried that and it doesn't work.

```
# Deliverfile
app_identifier 'com.xxx.app1036'
username 'support+3533@example.com'
metadata_path 'downloads/metadata/1036'
screenshots_path 'downloads/screenshots/1036'
force true
```

Still asks for:

`Do you want to overwrite existing metadata on path '/Users/barnacle/Documents/mytours/mytours-ios/downloads/metadata/1036'? (y/n)`

Any other settings I can try?

**Comment by barnaclebarnes** [comment]
Another datapoint. Setting `--force` on the command line updates the summary but still prompts for overwriting the files:

```
➜  mytours-ios git:(experiment/fastlane) ✗ FASTLANE_PASSWORD=xxx deliver download_metadata --force
[11:36:25]: Successfully loaded '/Users/barnacle/Documents/mytours/mytours-ios/fastlane/Deliverfile' 📄

+------------------+-----------------------------+
| Detected Values from './fastlane/Deliverfile'  |
+------------------+-----------------------------+
| app_identifier   | com.xxx.app1036      |
| username         | support+3533@example.com |
| metadata_path    | downloads/metadata/1036     |
| screenshots_path | downloads/screenshots/1036  |
+------------------+-----------------------------+

[11:36:25]: Login to iTunes Connect (support+3533@example.com)
[11:36:30]: Login successful

+-----------------------+-----------------------------+
|               deliver 1.15.0 Summary                |
+-----------------------+-----------------------------+
| force                 | true                        |
| app_identifier        | com.xxx.app1036      |
| username              | support+3533@example.com |
| metadata_path         | downloads/metadata/1036     |
| screenshots_path      | downloads/screenshots/1036  |
| skip_binary_upload    | false                       |
| skip_screenshots      | false                       |
| skip_metadata         | false                       |
| submit_for_review     | false                       |
| automatic_release     | false                       |
| dev_portal_team_id    | [[DEV_PORTAL_TEAM_ID]]      |
| overwrite_screenshots | false                       |
+-----------------------+-----------------------------+

Do you want to overwrite existing metadata on path '/Users/barnacle/Documents/mytours/mytours-ios/downloads/metadata/1036'? (y/n)
```

**Comment by TKBurner** [comment]
@barnaclebarnes What happens if you set up a lane with the _deliver_ command rather than calling the command directly from the command line? Thanks!

**Comment by barnaclebarnes** [comment]
@TKBurner Maybe a dumb question but how do you run the `download_metadata` action in the `deliver` command? i.e:

```
lane :download_metadata do
    app_id = "abc"
    deliver(force: true, download_metadata: true)
end
```

If I only want to download the metadata, what options do I send?

**Comment by TKBurner** [comment]
@barnaclebarnes Thanks for following up. This is not possible you can only sync the data between the two without an interactive shell. Sorry for the confusion.

**Comment by barnaclebarnes** [comment]
@TKBurner OK - So the issue still stands that `--force` true does not force overwriting of the metadata folder.

**Comment by barnaclebarnes** [comment]
I got it to work using this:

`DELIVER_FORCE_OVERWRITE=true deliver download_metadata`

Looks like it is not checking the `--force` argument or the `force` keyword in the file. Maybe just a documentation change is needed?

https://github.com/fastlane/fastlane/blob/652a1b3d0628572ae0bcece9a94b7db60fa16139/deliver/lib/deliver/commands_generator.rb#L130

**Comment by TKBurner** [comment]
@barnaclebarnes We'll look into this and see what's going on. Thanks!

**Comment by lacostej** [comment]
Addressed by #9307
